types/parse/parse-tests.ts

```diff
@@ -181,6 +185,12 @@ async function test_query_promise() {
     } catch (error) {
         // noop
     }
+
+    await getQuery.map((score, index) => score.increment("score", index));
+    await getQuery.reduce((accum, score, index) => accum += score.get("score"), 0);
```

Comment:
`+=` here only works since that also returns the new value, but you aren't actually modifying `accum` here

```suggestion
    await getQuery.reduce((accum, score, index) => accum + score.get("score"), 0);
```